modules/conflict_resolver/README.md

```diff
@@ -0,0 +1,53 @@
+# Conflict Resolver
+
+## Purpose
+
+The conflict resolver is intended to allow the resolution of conflicts
+(different data entry for a given field between the "single data
+entry" and "double data entry" variations) for instruments which
+have double data entry enabled.
+
+## Intended Users
+
+The conflict resolver is used by data entry staff.
```

Comment:
Might be good to add something about site coordinators and/or behavioural/imaging specialists.  Maybe lift a line from the Scope section about how it is not enforced by LORIS but add that the conflicts can be reviewed/resolved in the module by study members other than data entry staff?

Reply:
I added a paragraph